src/audio/smart_amp_test.c

```diff
@@ -146,8 +151,8 @@ static int smart_amp_ctrl_get_bin_data(struct comp_dev *dev,
 	case SOF_SMART_AMP_CONFIG:
 		ret = smart_amp_get_config(dev, cdata, size);
 		break;
-	case SOF_SMART_AMP_MODEL:
-		ret = comp_get_model(dev, &sad->model, cdata, size);
+	case SOF_SMART_AMP_DATA_BLOB:
```

Comment:
What's the rational to changing the macro name. Model is more meaningful than blob here. I think Maxim are using "model" too ?

Reply:
@lgirdwood 
I thought that it will be more clear. I returned to MODEL name. 
"I think Maxim are using "model" too" - do you mean specific "model" name?